### PR TITLE
feat(components/indicators): update alert test harness with accessibility functions

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/500/storybook/)

Component Storybooks:

- _(no component storybooks affected in this pr)_ 

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/500/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/500/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/500/playground/)

File: libs/components/indicators/testing/src/alert/alert-harness.ts [hunk 1]
```diff
@@ -42,11 +46,7 @@ export class SkyAlertHarness extends SkyComponentHarness {
       return 'success';
     }
 
-    if (await alert.hasClass('sky-alert-warning')) {
-      return 'warning';
-    }
-
-    return undefined;
+    return 'warning';
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/indicators/testing/src/alert/alert-harness.ts:49]: Can you explain why we are changing this? I know we have a default, but that default sets the class and without the class there is something wrong there and there is no alert type showing.

**Review by Blackbaud-TrevorBurch — COMMENTED** [review]
@Blackbaud-ErikaMcVey one question

File: libs/components/indicators/testing/src/alert/alert-harness.ts (hunk 1 above)
Comment (Blackbaud-ErikaMcVey) [reply libs/components/indicators/testing/src/alert/alert-harness.ts:49]: Warning is the default so there will never _not_ be a class--when we were returning undefined it was impossible to get test coverage to 100%.

If we break something to where the warning class no longer is added by default and no alert type is showing then I think it'd be the responsibility of the alert component tests to catch that, not the harness tests. What do you think?

Comment (Blackbaud-TrevorBurch) [reply libs/components/indicators/testing/src/alert/alert-harness.ts:49]: That’s fair. Ok.

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**